docs/src/main/sphinx/connector/cassandra.md

```diff
@@ -313,6 +313,11 @@ statements, the connector supports the following features:
 - {doc}`/sql/create-table-as`
 - {doc}`/sql/drop-table`
 
+### Procedures
+
+```{include} jdbc-procedures-execute.fragment
```

Comment:
Cassandra connector isn't JDBC-based connector. Using **_jdbc_**-procedures-execute.fragment is weird.

Reply:
but syntax and all is the same right? Should we maybe rename the fragment?

Reply:
I assume in general if we implement procedures we try to keep them the same across connectors .. jdbc or not .. so that might make sense anyway. Also for the flush one.

Reply:
> but syntax and all is the same right? Should we maybe rename the fragment?

Yes, the syntax is the same. I would recommend renaming the fragment.

Reply:
Done.